Package.swift

```diff
@@ -18,7 +18,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/pointfreeco/swift-composable-architecture", from: "0.9.0")
+        .package(url: "https://github.com/pointfreeco/swift-composable-architecture", .branch("main"))
```

Comment:
@igorkravchenko, is there a specific reason why to use the main branch instead of a tagged version?

Reply:
No, I probably need to pin it to specific exact version here to avoid breaking changes in the future.